modules/aws_k8s_base/tf_module/values-ha.yaml

```diff
@@ -30,55 +31,29 @@ publicAPIResources: *controller_resources
 identityResources:
   cpu: *controller_resources_cpu
   memory:
-    limit: 250Mi
+    limit: ""
```

Comment:
This means no limit on controller any longer?

Reply:
not needed for this

Reply:
much rather see it take more than expected memory than fail